src/main/java/com/pokegoapi/auth/GoogleAuthCallback.java

```diff
@@ -0,0 +1,8 @@
+package com.pokegoapi.auth;
```

Comment:
It could. Would be looking like this: https://github.com/FabianTerhorst/PokeGOAPI-Java/commit/5759c0ee3787e61c56ea9997dfd413ec47445edf